plinth/modules/sso/__init__.py

```diff
@@ -0,0 +1,46 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Plinth module to configure Single Sign On services.
+"""
+
+from plinth import action_utils
+from django.utils.translation import ugettext_lazy as _
+
+version = 1
```

Comment:
I'm wondering how the module upgrades can work. Currently, modules are only upgraded when the user visits a page for the module, and they are redirected to a setup screen. But this doesn't seem to work for sso module, which doesn't have its own page.